commcare_connect/templates/opportunity/opportunity_list.html

```diff
@@ -89,53 +89,55 @@ <h1> Opportunities
-                <span aria-hidden="true">&laquo;</span>
-              </a>
-            </li>
+{% if opportunity_invitations %}
+<div class="container bg-white shado mb-5">
+  <div class="mt-5 py-3">
+    <h2>Opportunity Invitations</h2>
+  </div>
+  <div class="pb-4">
+    <table class="table border table-responsive">
+      <thead class="table-light">
+      <tr>
+        <th>Name</th>
+        <th>Start Date</th>
+        <th>End Date</th>
+        <th>Status</th>
+        <th>Manage</th>
+      </tr>
+      </thead>
+      <tbody>
+      {% for invitation in opportunity_invitations %}
+      <tr>
+        <td>{{ invitation.managed_opportunity.name }}</td>
+        <td>{{ invitation.managed_opportunity.start_date|default:"Not Set" }}</td>
+        <td>{{ invitation.managed_opportunity.end_date|default:"Not Set" }}</td>
+        <td>
+          {% if invitation.managed_opportunity.is_setup_complete %}
+          {% if invitation.managed_opportunity.is_active %}
+          <span class="badge bg-success rounded-pill">Active</span>
           {% else %}
-            <li class="page-item disabled">
-              <a class="page-link">
-                <span aria-hidden="true">&laquo;</span>
-              </a>
-            </li>
+          <span class="badge bg-secondary text-white rounded-pill">Inactive</span>
           {% endif %}
-
-          {% for page_number in page_obj.paginator.page_range %}
-            {% if page_obj.number == page_number %}
-              <li class="page-item active">
-                {% else %}
-              <li class="page-item">
-            {% endif %}
-          <a class="page-link" href="?{% update_query_params page=page_number %}">
-            {{ page_number }}
-          </a>
-          </li>
-          {% endfor %}
-
-          {% if page_obj.has_next %}
-            <li class="page-item">
-              <a class="page-link"
-                 href="?page={{ page_obj.next_page_number }}" aria-label="Next">
-                <span aria-hidden="true">&raquo;</span>
-              </a>
-            </li>
           {% else %}
-            <li class="page-item disabled">
-              <a class="page-link">
-                <span aria-hidden="true">&raquo;</span>
-              </a>
-            </li>
+          <span class="badge bg-warning text-white rounded-pill">Pending Setup</span>
           {% endif %}
-        </ul>
-      </nav>
-    {% endif %}
+        </td>
+        <td width="300">
+          <div>
+            <a class="btn btn-primary btn-sm"
+               href="{% url 'opportunity:apply_opportunity_invite' org_slug=request.org.slug pk=invitation.managed_opportunity.id application_id=invitation.id %}">
+              <span class="bi bi-check-circle-fill"></span><span class="d-none d-md-inline">&nbsp;Apply</span>
+            </a>
+          </div>
+        </td>
+      </tr>
+      {% endfor %}
+      </tbody>
+    </table>
```

Comment:
strong plus one to this

Reply:
I have implemented this in all the other views, but this one is a bit tricky because there is already a table (Opportunity list) that includes sorting. If I create another table with Django-tables2, the pagination of the first table will affect the second one. I think I might need to change the entire view to implement Django-tables2. Should I do that?

Reply:
I think we are fine with leaving the Opportunity List table for now.